### ISSUE TITLE
[Source Engine] Translucent sprites rendering issue

### ISSUE THREAD

**Issue by yurakone** [doitsujin/dxvk#4006]
Hard edges on translucent sprites `env_sprite` with `World Space Glow` render rmode `(9)` in Source engine games, also affects to `env_lensflare` in Black Mesa

### Software information
Tested on Half-Life 2 Deathmatch, Black Mesa

### System information\Tested on
- GPU: NVIDIA GeForce RTX 3060 Ti \ NVIDIA GeForce GTX 1660 Ti
- OS: Windows 10 / 11
- Driver: NVIDIA 552.22.0
- DXVK version: 32bit since 1.3.9 (earlier versions have not been tested) to 2.3.1

### Apitrace file(s)
[hl_trace](https://drive.google.com/file/d/1brCN_jd_Ro-xITbn6pcp3Khai8s7Sbnw/view?usp=sharing)

### Log files
[hl2_d3d9.log](https://github.com/doitsujin/dxvk/files/15357953/hl2_d3d9.log)

### Screenshots
Black Mesa sreenshots:
![IMG_8029](https://github.com/doitsujin/dxvk/assets/98318619/d2abc917-33aa-4fc4-b34c-d23c37649797)
![IMG_8030](https://github.com/doitsujin/dxvk/assets/98318619/39af463c-980d-4922-97b6-f69186ae0393)
Half-Life 2 Deathmatch sreenshots:
![IMG_8031](https://github.com/doitsujin/dxvk/assets/98318619/034fa9e3-4c0e-48af-be1b-dddbb10c2175)
![IMG_8032](https://github.com/doitsujin/dxvk/assets/98318619/b1586324-f378-4d0c-9597-f2e1d1b43bf2)

**Comment by Blisto91** [comment]
Just checking out the apitrace it looks fine on my RX 7900xtx with both radv and amdvlk

**Comment by K0bin** [comment]
![I can reproduce it](https://github.com/doitsujin/dxvk/assets/1131720/1353d550-aeb4-442e-b699-bd5fa4170094)

**Comment by K0bin** [comment]
Internal note:
It's blending a texture that gets uploaded via a simple Lock,Copy,Unlock onto the render target. The blending is a simple add and output alpha of the fragment shader is 1.0 across the board.
The only thing that might be interesting is that both the RT as well as the texture are reinterpreted as SRGB.

**Comment by yurakone** [comment]
I looked into the problem a little. I added alpha channel to the texture, cuz it was with no alpha `DXT1` and created a new material then I tried different shader parameters
--Shader used: `UnlitGeneric`
--Shader parameters tested: `$translucent 1`  `$additive 1`
--Texture format: `DXT5 Compressed texture`
![20240519015728_1](https://github.com/doitsujin/dxvk/assets/98318619/15e3b631-e36d-40c7-905e-d7fade13e1c2)
![20240519015828_1](https://github.com/doitsujin/dxvk/assets/98318619/3693c11f-9b67-417d-86ef-549818e29291)
So... this looks like a problem with `$additive 1` only, and maybe Nvidia drivers

**Comment by Blisto91** [comment]
This indeed is looking like a Nvidia driver bug. We've confirmed that it looks fine on AMD with the radv and amdvlk driver. On Intel it looks good with the Linux driver and on Nvidia it also looks fine on the NVK Linux driver while Nvidia own have the issue.

**Comment by Blisto91** [comment]
No. A bug report needs to be made to Nvidia, which we haven't gotten to yet (I've volunteered now)

**Comment by Blisto91** [comment]
Forgot to update here that this have been reported to Nvidia and is being tracked as bug number 4762602

**Comment by Blisto91** [comment]
@yurakone Are you able to verify that this now seems fine with 576 on Windows or 575 on Linux?

**Comment by yurakone** [comment]
Now it looks fine, 576 Windows 
![Image](https://github.com/user-attachments/assets/d1f4c0f4-b9dd-4396-a207-24b2cb4e513c)
![Image](https://github.com/user-attachments/assets/c6b94948-4898-4e31-a00c-d94bb163c927)
